gatsby-node.js

```diff
@@ -9,6 +9,8 @@ const axios = require('axios')
 
 const BASE_URL = 'http://localhost:5000/fetch?url='
```

Comment:
would prefer this to be rewritten as a function like const PROXY_URL = url => `http://localhost:5000/fetch?url=${url}`. This will make the intended use more clear

Reply:
also would prefer this moved into a constants file